src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute is linking to a SCSS file (`styles/main.scss`). HTML cannot directly use SCSS files. You need to compile the SCSS into a CSS file and link to that CSS file instead.